ui/imports/shared/status/StatusGifPopup/ConfirmationPopup.qml

```diff
@@ -85,8 +85,4 @@ Popup {
             root.close()
         }
     }
-
-    onClosed: {
-        root.destroy();
```

Comment:
This one broke the GIF popup; the "Enable" overlay was just gone and the user couldn't enable GIFs as a result :/